routers/web/web.go

```diff
@@ -757,6 +757,7 @@ func RegisterRoutes(m *web.Route) {
 				m.Post("/reactions/{action}", bindIgnErr(forms.ReactionForm{}), repo.ChangeIssueReaction)
 				m.Post("/lock", reqRepoIssueWriter, bindIgnErr(forms.IssueLockForm{}), repo.LockIssue)
 				m.Post("/unlock", reqRepoIssueWriter, repo.UnlockIssue)
+				m.Post("/delete", reqRepoIssueWriter, repo.DeleteIssue)
```

Comment:
```suggestion
				m.Post("/delete", reqRepoAdmin, repo.DeleteIssue)
```
This would match the API permissions and seems more sane at first glance, though it's arguable who should be able to delete issues.

Reply:
Yes, thank you. I tried copying it from the other router, but it has brackets there. It thus didn't work out, so I stuck to something I found it that router.